test/on_yubikey/test_cli_piv.py

```diff
@@ -167,6 +152,144 @@ def test_export_attestation_certificate(self):
         self.assertIn('BEGIN CERTIFICATE', output)
 
 
+class GenerateSelfSigned_NonDefaultMgmKey(PivTestCase):
```

Comment:
Not a fan of underscore in classnames.

Reply:
I agree, but I think it makes a useful visual separator in this case. Do you prefer this

```
test_generate_self_signed_slot_9a (test.on_yubikey.test_cli_piv.GenerateSelfSigned_NonDefaultMgmKey) ... ok
test_generate_self_signed_slot_9c (test.on_yubikey.test_cli_piv.GenerateSelfSigned_NonDefaultMgmKey) ... ok
test_generate_self_signed_slot_9d (test.on_yubikey.test_cli_piv.GenerateSelfSigned_NonDefaultMgmKey) ... ok
test_generate_self_signed_slot_9e (test.on_yubikey.test_cli_piv.GenerateSelfSigned_NonDefaultMgmKey) ... ok
test_generate_self_signed_slot_9a (test.on_yubikey.test_cli_piv.GenerateSelfSigned_ProtectedMgmKey) ... ok
test_generate_self_signed_slot_9c (test.on_yubikey.test_cli_piv.GenerateSelfSigned_ProtectedMgmKey) ... ok
test_generate_self_signed_slot_9d (test.on_yubikey.test_cli_piv.GenerateSelfSigned_ProtectedMgmKey) ... ok
test_generate_self_signed_slot_9e (test.on_yubikey.test_cli_piv.GenerateSelfSigned_ProtectedMgmKey) ... ok
```
or this?
```
test_generate_self_signed_slot_9a (test.on_yubikey.test_cli_piv.GenerateSelfSignedWithNonDefaultMgmKey) ... ok
test_generate_self_signed_slot_9c (test.on_yubikey.test_cli_piv.GenerateSelfSignedWithNonDefaultMgmKey) ... ok
test_generate_self_signed_slot_9d (test.on_yubikey.test_cli_piv.GenerateSelfSignedWithNonDefaultMgmKey) ... ok
test_generate_self_signed_slot_9e (test.on_yubikey.test_cli_piv.GenerateSelfSignedWithNonDefaultMgmKey) ... ok
test_generate_self_signed_slot_9a (test.on_yubikey.test_cli_piv.GenerateSelfSignedWithProtectedMgmKey) ... ok
test_generate_self_signed_slot_9c (test.on_yubikey.test_cli_piv.GenerateSelfSignedWithProtectedMgmKey) ... ok
test_generate_self_signed_slot_9d (test.on_yubikey.test_cli_piv.GenerateSelfSignedWithProtectedMgmKey) ... ok
test_generate_self_signed_slot_9e (test.on_yubikey.test_cli_piv.GenerateSelfSignedWithProtectedMgmKey) ... ok
```

...and it's "just" the tests, not part of any API. :)

Reply:
I prefer the second version just because my brain reacts to the mixing of styles, but don't have very strong opinions.

Reply:
Perhaps I should move those tests to a separate module instead...